module/grpcserver/interceptor_logging.go

```diff
@@ -51,3 +38,16 @@ func InterceptorLogger(l zerolog.Logger) logging.Logger {
 		}
 	})
 }
+
+func customClientCodeToLevel(c codes.Code) logging.Level {
```

Comment:
can we add a comment here or rename it. Just calling it custom doesn't actually tell you why/when its needed. I checked and this seems to be for all GRPC calls, but the DefaultServerCodeToLevel also mentions its for GRPC calls, so why is there a custom one needed here?